src/treeChooser/treeChooser.controller.js

```diff
@@ -598,7 +599,7 @@ function TreeChooserController(
     // Default filter node function
     if (!_.isFunction(vm.filterNode)) {
       vm.filterNode = function (item, filterText) {
-        return _.includes(_.toLower(_.get(item, vm.properties.label)), _.toLower(filterText));
+        return _.includes(_.toLower(_.get(item, vm.properties.listLabel)), _.toLower(filterText));
```

Comment:
Will listLabel always contain what's in label plus some additional text? If not, we should edit the search to search by listLabel and label

Reply:
In my opinion you're right. But is it possible for me to make changes in next PR? On this week, but after this.

Reply:
It's updated, please look.